test/TestProjects/SubscriptionExtensions/Generated/Configuration.json

```diff
@@ -16,6 +16,9 @@
   "CredentialScopes": [],
   "CredentialHeaderName": "api-key",
   "OperationGroupToResourceType": {},
-  "OperationGroupToResource": {},
+  "OperationGroupToResource": {
+    "AvailabilitySets": "NonResource",
```

Comment:
Can we rename these two things

Toaster
Oven

AvailabilitySet and VirtualMachine are real things and it throws me off that they are NonResource

Reply:
Proposals ?

- null
- ""
- "N/A"
- "None"